SwiftUIDZ/ModelView/DetailViewModel.swift

```diff
@@ -11,11 +11,14 @@ final class DetailViewModel: ObservableObject {
         static var emptyText = ""
         static var minTextNumber = 0
         static var maxTextNumber = 100
+        static var scaleNumber: CGFloat = 1
     }
 
     // MARK: - Public Properties
 
     @Published var lastDescriptionText = Constants.emptyText
+    @Published var scale: CGFloat = Constants.scaleNumber
```

Comment:
```suggestion
    @Published var scale: CGFloat = 11 например
```
вынести в константы что б присвоить свойству?